src/FredApi/Dockerfile

```diff
@@ -1,17 +1,17 @@
 # minimized Dockerfile for a .Net Core sample Dockerfile based on the Alpine Image
```

Comment:
@swolg Optional: Die Dockerfiles könnten wir noch ein bisschen tunen. Details hier: https://medium.com/01001101/containerize-your-net-core-app-the-right-way-35c267224a8d

Reply:
Hab mich and dem "Smaller, smaller, smaller" Paragraphen deines Blogposts orientiert.

Reply:
@Chris1995x ok. Das ist eher für Edge cases. Der Standard wäre "A good starting point". Aber entscheidet ihr.